plugins/DataLoadULog/ulog_parser.cpp

```diff
@@ -736,10 +748,15 @@ ULogParser::Timeseries ULogParser::createTimeseries(const ULogParser::Format* fo
     {
         for( const auto& field: format.fields)
         {
+            // skip padding messages
+            if (field.field_name == "_padding0") {
```

Comment:
You should skip all fields starting with `_padding`, see https://dev.px4.io/en/log/ulog_file_format.html#definitions-section

Reply:
@bkueng Thanks, will do!

Reply:
Done

Reply:
I guess you forgot that one?

Reply:
I took care of it

Reply:
Thank you.